src/components/shapes/helpers.js

```diff
@@ -279,3 +283,19 @@ function convertPath(options, x2p, y2p) {
         return segmentType + paramString;
     });
 }
+
+function getPixelShift(axis, shift) {
+    shift = shift || 0;
+    var shiftPixels = 0;
+    if(axis && shift) {
+        var isVertical = axis._id.charAt(0) === 'y';
+        if(axis.type === 'category' || axis.type === 'multicategory') {
+            if(isVertical) {
```

Comment:
Please test these on reversed x and y axes too.
Thank you!

Reply:
thx, it didn't actually work and the fix resulted in simpler code.